ert_shared/cli/main.py

```diff
@@ -18,6 +21,18 @@
 from ert_shared.libres_facade import LibresFacade
 from res.enkf import EnKFMain, ResConfig
 
+if sys.version_info < (3, 7):
+    from async_generator import asynccontextmanager
+else:
+    from contextlib import asynccontextmanager
+
+
+@asynccontextmanager
+async def start_experiment_server(ee_config: EvaluatorServerConfig):
```

Comment:
Naive question :) Why do we need this as a contextmanager function as nothing happens after we yield?